beta-src/src/utils/map/writeBuildNotifications.ts

```diff
@@ -0,0 +1,87 @@
+import { current } from "@reduxjs/toolkit";
+import { GameState } from "../../state/interfaces/GameState";
+
+const defaultStyle = {
+  borderRadius: "8px",
+  backgroundColor: "rgba(0,0,0,.4)",
+  color: "#ffffff",
+  fontSize: "16px",
+  height: 40,
+  lineHeight: 2.5,
+  m: 1,
+  textAlign: "center",
+};
+
+// Currently, indexes 0 and 1 are being used as the delete labels.
+// 0 is the tracker which tells the player how many units need to be destroyed and is always displayed
+// when needing to destroy a unit and 1 renders conditionally based on whether the player has
+// selected any units so far or not
+
+/* eslint-disable no-param-reassign */
+export default function writeBuildNotifications(state): void {
+  const {
+    ordersMeta,
+    overview: { phase },
+  }: {
+    // definition
+    ordersMeta: GameState["ordersMeta"];
+    overview: { phase: GameState["overview"]["phase"] };
+  } = current(state);
+  if (phase === "Builds") {
+    let ordersToGo = 0;
+    let totalOrders = 0;
+    let toDo;
+    // values instead of entries key/value difference
+    Object.values(ordersMeta).forEach(({ update }) => {
+      if (update?.toTerrID === null) ordersToGo += 1;
+      totalOrders += 1;
+      if (update?.type === "Destroy") toDo = "Destroy";
+    });
+    if (toDo === "Destroy") {
+      if (ordersToGo > 1) {
+        state.notifications[0] = {
+          message: `Select ${ordersToGo} units to destroy.`,
+          style: {
+            ...defaultStyle,
+            fontWeight: 700,
+            width: 200,
+          },
+        };
+      } else if (ordersToGo === 1) {
+        state.notifications[0] = {
+          message: "Select 1 unit to destroy.",
+          style: {
+            ...defaultStyle,
+            fontWeight: 700,
+            width: 200,
+          },
+        };
+      } else if (ordersToGo === 0) {
+        state.notifications[0] = {
+          message: "Orders ready to submit",
+          style: {
+            ...defaultStyle,
+            fontWeight: 700,
+            width: 200,
```

Comment:
width and fontWeight can be part of defaultStyle and just override it on L74 and L75